gdk4-wayland/src/auto/versions.txt

```diff
@@ -1,2 +1,2 @@
-Generated by gir (https://github.com/gtk-rs/gir @ 56b95ac)
-from gir-files (https://github.com/gtk-rs/gir-files @ 6088bb6)
+Generated by gir (https://github.com/gtk-rs/gir @ 4e4ba32)
+from gir-files (@ 6088bb6)
```

Comment:
Seems like this doesn't work as it should @MarijnS95 ?

Reply:
Strange indeed, it works fine for GStreamer... Let me check `.gitmodules` here...

Reply:
Should be fixed with https://github.com/gtk-rs/gir/pull/1090.

Reply:
Seems to be working now! Thanks @MarijnS95 :D

Reply:
Yup, looks like we didn't strip `.git` though. As with all things I keep repeating this one sentence: "GStreamer-rs doesn't have it, didn't notice" :grimacing: 

(Though over at `GStreamer-rs`  we _do_ have `.git` for the fd.o-hosted `gst-gir-files` repo because GitLab prints annoying redirects)

Anyway, this regen doesn't make any difference because `docs/` aren't checked in; maybe you want to wait with the comment-regeneration noise until something interesting is merged to `gir`?